### DISCUSSION TITLE
Playlist Sorting

### DISCUSSION THREAD

**Discussion by Vulqci** [ciderapp/Cider#1515] [Feature Request]
Just a simple feature request - the ability to change how your playlists are sorted would be nice. For example, sorting by title, artist, release date, or change playlist order sort (put new music at the top of the playlist instead of the bottom), etc.

**Comment by Trend-Net** [comment]
Also the option to move songs up and down in the playlist would be nice ^^

**Comment by scwood** [comment]
I also would like the ability to sort by added most recently. For example I have a "favorites" type playlist and I usually listen to the newest songs first in both iTunes and Spotify.

**Comment by brewera** [comment]
I think this would be a great feature.  I use the Shazam application and it adds music to the "My Shazam Tracks" playlist automatically but the newest songs are at the bottom.
